### DISCUSSION TITLE
--extra-models-cpu | anyone found a workaround to make it work again?

### DISCUSSION THREAD

**Discussion by dudumaroja** [hlky/stable-diffusion-webui#1085] [General]
I cant say witch revision ago, something broke in gfpgan stopped working in CPU.. just on gpu.. someone has a mirror before this change?

i have a gtx 1660 ti 6gb and used to work amazing well with --optimizedturb --extra-models-cpu --precision full --no-half 

few days ago for windows shananingans reasons need to format and reinstall everthing again.. but now  --optizedturbo works only if i dont even look to gfpgan..  and --optimzed sometimes works.. sometimes i get cuda OOM errors.. just a lotery..

[its sad becouse a week ago i cound even generate larger then 512x512 files and using extra models and optimizedturbo.. ](https://github.com/sd-webui/stable-diffusion-webui/discussions/693)

i need to find a old backup :(

**Comment by deniansouza** [comment]
I also have a 1660ti 6GB and GFPGAN is not working with --extra-models-cpu. RealESRGAN works though.
